.github/workflows/ci.yaml

```diff
@@ -9,22 +9,30 @@ on:
 
 jobs:
 
-  test:
+  build:
```

Comment:
I hadn't dug into this yaml file previously, but it looks like we're triggering on push and pull requests to the main branch. Does that mean any PR will trigger the full CI? It looks like we have a build and release below, that may not be what we want